### PR TITLE
Bug and issue fixes (3)

### PR DESCRIPTION
**Pull request by sg00dwin**
Fix log status text on IE
	https://bugzilla.redhat.com/show_bug.cgi?id=1277039

Wrap non breaking strings correctly in FF
	https://github.com/openshift/origin/issues/5542

Give spacing on narrow width tables for status
	https://github.com/openshift/origin/issues/5621

### CODE REVIEW

**Comment by sg00dwin** [comment]
Fixes https://github.com/openshift/origin/issues/5542
FF
screen shot 2015-11-03 at 1 33 47 pm

Fixes bug https://bugzilla.redhat.com/show_bug.cgi?id=1277039
IE
screen shot 2015-11-03 at 1 55 51 pm

**Comment by sg00dwin** [comment]
@jwforres @spadgett

File: assets/app/styles/_log.less [hunk 1]
```diff
@@ -99,18 +100,19 @@
...
   }
   div {
-    width: 60px;
     padding-right: 10px;
   }
 }
 .log-line-text {
   padding: 0 10px;
   white-space: pre-wrap;
+  .word-break();
```
Comment (spadgett) [inline assets/app/styles/_log.less]: @sg00dwin Is this going to break words in the middle always?

File: assets/app/styles/_log.less [hunk 2]
```diff
@@ -99,18 +100,19 @@
   }
 }
 .log-line-number {
+  .flex(@columns: 0 0 60px); // set width of number column
```
Comment (spadgett) [inline assets/app/styles/_log.less]: Is that width enough to accommodate 5 digits?

File: assets/app/styles/_core.less [hunk 3]
```diff
@@ -690,6 +690,7 @@ a.action-button {
 }
 
 .status {
+  min-width: 130px;
```
Comment (spadgett) [inline assets/app/styles/_core.less]: Wide enough for the word "Terminated"?

File: assets/app/styles/_log.less (hunk 2 above)
Comment (sg00dwin) [inline assets/app/styles/_log.less]: yes
screen shot 2015-11-03 at 3 42 07 pm

File: assets/app/styles/_core.less (hunk 3 above)
Comment (sg00dwin) [inline assets/app/styles/_core.less]: Actually, this is for an IE issue I saw in the table list pages where using <div flex> within a <td> has the side effect of collapsing the <td> aggressively. By setting the min-width:130px the div alleviates that.

**with min-width:130px**
with130


**without min-width:130px**
without130

Comment (spadgett) [inline assets/app/styles/_core.less]: Ah ha, OK. Thanks.

File: assets/app/styles/_log.less (hunk 1 above)
Comment (sg00dwin) [inline assets/app/styles/_log.less]: No - Chrome and Safari  (they support ```word-break: break-word```)
Yes - Firefox and IE (they will use ```word-break:break-all```)

Comment (spadgett) [inline assets/app/styles/_log.less]: It looks like Firefox supports `word-wrap: break-word;`, which would avoid breaking in the middle unless we have to. Is it possible to use that instead?

**Comment by jwforres** [comment]
LGTM @openshift/team-project-committers  probably on the bubble, but fixes several gross visual issues.  has been tested across browsers already

**Comment by smarterclayton** [comment]
Approved but I'm going to put this right at the end.

**Comment by smarterclayton** [comment]
[test]

**Comment by smarterclayton** [comment]
[merge]

**Pull request merged**